addons/hr_timesheet/models/project_project.py

```diff
@@ -10,7 +10,7 @@ class Project(models.Model):
 
     allow_timesheets = fields.Boolean(
         "Timesheets", compute='_compute_allow_timesheets', store=True, readonly=False,
-        default=True)
+        default=True, export_string_translation=False)
```

Comment:
This field seems display in the UI and so I don't think it is a good idea to not translate its string. 

What do you think? 🧐